tests/conftest.py

```diff
@@ -192,6 +192,12 @@ def sim_6363_device(system: nidaqmx.system.System) -> nidaqmx.system.Device:
     return _device_by_product_type("PCIe-6363", DeviceType.SIMULATED, system)
 
 
+@pytest.fixture(scope="function")
+def sim_6535_device(system: nidaqmx.system.System) -> nidaqmx.system.Device:
+    """Gets simulated 6535 device information."""
+    return _device_by_product_type("PCIe-6535", DeviceType.SIMULATED, system)
```

Comment:
FYI - this will automatically `pytest.skip` if it can't find the device, which means we're good on Linux. We won't have coverage there, but not much we can do about that.